### PR TITLE
Clarify migration script dependencies and make aware of the mysql_connector_python error

### PR DESCRIPTION
**Pull request by marwyg**
I updated the docs, so people hopefully don't run into the weird "UTF-8" error, when executing the 11-to-12 database migration script.

### CODE REVIEW

**Review by mliradelc — APPROVED** [review]
Good for me

File: docs/guides/admin/docs/upgrade.md [hunk 1]
```diff
@@ -49,10 +49,19 @@ You can do this in a virtual Python environment if you do not want to install th
...
 pip install psycopg2-binary
-# or for MariaDB
+
+# or for MariaDB:
 pip install mysql-connector-python
+# NOTE: please don't install version 8.0.30, because the migration
+# script runs into errors with this version. Please install
+# version 8.0.29 or try a newer version than 8.0.30 if it exists. 
+# You can install a specific version with: 
+# 'pip install mysql_connector_python==8.0.29'
```
Comment (lkiesow) [inline docs/guides/admin/docs/upgrade.md]: Why not just recommend installing that version?

```suggestion
# or for MariaDB:
pip install mysql-connector-python==8.0.29'
```

**Review by lkiesow — COMMENTED** [review]
The documentation in the script still tells you to install the latest version:

https://github.com/opencast/opencast/blob/bb76ce70bd7e695a4a132c279e3ed629818c52d1/docs/upgrade/11_to_12/workflows_mariadb.py#L3-L4

**Comment by marwyg** [comment]
I made two PRs: https://github.com/opencast/opencast/pull/4033 and to be honest, I have no idea why I thought, this is a good idea. So I added both doc changes into this PR. I also changed the installation command, so you are always installing version 8.0.29 of the module.

**Comment by marwyg** [comment]
https://github.com/opencast/opencast/pull/4033#issuecomment-1198995360
@mliradelc I tried to put a try-catch around the connection, but had trouble with it. Could you add it, like you suggested it in the other PR?

**Comment by mliradelc** [comment]
@marwyg That should make it 😄

**Comment by marwyg** [comment]
Thank you :grin: :+1:

**Pull request merged**